src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -722,15 +723,15 @@ static bool list_item_sort(LoadSaveListItem &a, LoadSaveListItem &b)
     switch (gConfigGeneral.load_save_sort)
     {
     case SORT_NAME_ASCENDING:
-        return strcicmp(a.name, b.name) < 0;
+        return a.name.compare(b.name) < 0;
```

Comment:
You have lost the case insensitive comparison.

Reply:
Ah, you're right. Shall I revert it to use Util.h's `strcicmp` using C-strings, then?